dbqueries.py

```diff
@@ -27,28 +27,36 @@ class StandardQueryData(QueryData):
 
 
 class TableCountsQueryData(QueryData):
-    type: Literal['table_counts']
     tables: list[str]
 
 
+class RecordCountsQueryData(TableCountsQueryData):
```

Comment:
maybe rename this to `TableRecordCounts`, to be in consistent with the actual string value `table_record_counts `?

Reply:
I can probably clean this up a bit; before I was using `TableCountsQueryData` as a base class for two different types, but that's unnecessary now with the refactor.

Reply:
Done. See https://github.com/tl-its-umich-edu/unizin-validation/pull/34/commits/7ff169d850c545c597690efbe6861dbcaab408d3